lib/rules/sort-destructuring-keys.js

```diff
@@ -54,6 +56,65 @@ const getOrder = ( order, modifier ) => {
 const getKeyName = node =>
   node.type === 'ExperimentalRestProperty' ? node.argument.name : node.key.name
 
+/**
+ * List all identifiers' name for given node
+ *
+ * @param {ASTNode} node - The property node.
+ * @returns {Array<String>} identifiers - List of identifiers
+ */
+const getIdentifiers = node => {
```

Comment:
Move this to a separate file in `helpers` so we can remove the `eslint-disable` comment at the top of the file.